.github/workflows/build.yml

```diff
@@ -75,20 +71,24 @@ on:
         type: string
         default: false
         required: false
-      repeat:
-        type: string
-        default: false
       build_name:
         type: string
         required: false
 
+
+env:
+  tag: ${{ (inputs.dev_image == 'true' || inputs.dev_image == true) && format('dev-{0}', github.head_ref || github.ref_name) || 'latest' }}
+  build_name: ${{ inputs.build_name || github.head_ref || github.ref_name }}
+  version: ${{ github.event.inputs.version && format('--version {0}', github.event.inputs.version) || '' }}
+
 jobs:
   health_check:
     name: Health Check
     runs-on: ubuntu-latest
     outputs:
-      tag: ${{ (inputs.dev_image == 'true' || inputs.dev_image == true) && format('dev-{0}', github.head_ref || github.ref_name) || 'latest' }}
-      build_name: ${{ inputs.build_name || github.head_ref || github.ref_name }}
+      plan_command: recipe --recipe-path ./${{ inputs.recipe_file }}.yml ${{ env.version }}
+      tag: ${{ env.tag }}
```

Comment:
env vars cannot be passed to workflow_calls (they're just not accessible at that part of the workflow), so they need to be here. But I thought it's a bit cleaner to have them defined up higher just to make them a bit clearer.

However, the "plan_command" had to be defined here because it depends on those vars